rbac/server/db/users_query.py

```diff
@@ -80,6 +80,20 @@
         raise ApiNotFound("Not Found: No user with the id {} exists".format(next_id))
 
 
+async def delete_user_resource(conn, next_id):
+    """Database query to delete an individual user."""
+    resource = (
+        await r.table("users")
```

Comment:
So when you delete a user from rethinkDB you need to remove it from all tables it may be in. This includes role_owners, role_admins, role_members, auth, user_mapping, pack_owners, and task tables as well (Even though they are not currently implemented.   Each one of these tables have the user removed if they exist otherwise it is an incomplete deletion and will cause issues when displaying portions of the app